README.rst

```diff
@@ -51,7 +51,7 @@ save it as a file called ``config.json``:
         "web_path": "/api",
         "web_port": 8051,
         "noisy": true,
-        "app_id": "YOUR_FB_APP_ID",
+        "app_id": "YOUR_FB_PAGE_APP_ID",
```

Comment:
We may as well choose to rename this to `page_id` instead of `app_id`. The configuration model allows for falling back to a previously used configuration name if the new name isn't provided, see: http://confmodel.readthedocs.org/en/latest/advanced.html#field-fallbacks